pytorch_to_returnn/import_wrapper/context.py

```diff
@@ -118,12 +121,28 @@ def _raise_not_implemented(*args):
     torch.nn.Module: ExplicitWrappedType(torch.nn.Module, WrappedModuleBase, wrap=_raise_not_implemented),
   }
 
+  class Functional(WrappedModuleBase):
+    def __init__(self, func, func_name):
+      super(Functional, self).__init__()
+      self.func = func
+      self.func_name = func_name
+
+    def forward(self, *args, **kwargs):
+      return self.func(*args, **kwargs)
+
+    @classmethod
+    def has_torch_forward(cls) -> bool:
```

Comment:
Why do you need this?

Reply:
Without this, the namespace is flattened according to `_flatten_namespace_for_mod` and the entry is not correctly registered in `make_module_call`.

Reply:
I don't exactly understand.

`_flatten_namespace_for_mod` will flatten when `has_torch_forward()` returns False, which is what you do here. So the opposite of what you say?

Also, why does `_flatten_namespace_for_mod` matter here?

But anyway, maybe add such a comment into the code.

Reply:
With the change here, `mod.has_torch_forward()` is `False`, so `_flatten_namespace_for_mod` will return `False`.

This matters in `make_module_call`:

```
    if parent_namespace is root_namespace and _flatten_namespace_for_mod(module_entry):
      # Special case, somewhat nicer to flatten the namespace for this case.
      root_namespace.assign_call(entry)
      namespace = root_namespace
    else:
      namespace = parent_namespace.register_sub_call(entry)
```

Only in the second branch, a corresponding entry in the namespace's `childs_by_name` is created which we need lateron.

Reply:
Yes, I'll add a comment.